src/EnergyPlus/SimAirServingZones.hh

```diff
@@ -239,17 +239,6 @@ struct SimAirServingZonesData : BaseGlobalStruct
     Real64 Vou = 0.0;            // Uncorrected outdoor air intake for all zones per ASHRAE std 62.1
     Real64 Vot = 0.0;            // Required outdoor air intake at primary AHU per ASHRAE std 62.1
 
-    Array1D_int CtrlZoneNumsCool;
```

Comment:
Removed these temporary arrays from `state` - they are used only in `InitAirLoops`.

Reply:
Yup